dokka-runners/runner-gradle-plugin-classic/src/test/kotlin/org/jetbrains/dokka/gradle/AndroidAutoConfigurationTest.kt

```diff
@@ -14,22 +14,27 @@ import kotlin.test.*
 
 class AndroidAutoConfigurationTest {
 
-    private val project = ProjectBuilder.builder().build().also { project ->
-        project.plugins.apply("com.android.library")
-        project.plugins.apply("org.jetbrains.kotlin.android")
-        project.plugins.apply("org.jetbrains.dokka")
-        project.extensions.configure<LibraryExtension> {
-            compileSdkVersion(28)
+    // Lazy because it throws an error if an Android test is run under Java 8
+    private val project by lazy {
+        ProjectBuilder.builder().build().also { project ->
+            project.plugins.apply("com.android.library")
+            project.plugins.apply("org.jetbrains.kotlin.android")
+            project.plugins.apply("org.jetbrains.dokka")
+            project.extensions.configure<LibraryExtension> {
+                compileSdkVersion(28)
+            }
         }
     }
 
     @Test
+    @Ignore("Current version of AGP requiers Java 11. TODO: if expected, disable this test for Java 8")
```

Comment:
AGP 7.1.3 (which is a requirement for KGP 2.0.0) requires Java 11 to run. I've asked the build tools team if it's intentional, and if it is - we can just mute these tests when running under Java 8, I guess. Will come back to it

Reply:
any news here?

Reply:
Yep! AGP 7.1.3+ is compiled with Java 8, but requires Java 11 to actually run. So we are compatible, but need to skip these tests with Java < 11. Will use the same solution as https://github.com/Kotlin/dokka/blob/d4d457d357627fd8100e4f0d47868020485996ec/dokka-subprojects/analysis-kotlin-api/src/test/kotlin/org/jetbrains/dokka/analysis/test/documentable/ObviousFunctionsTest.kt#L124-L136